mobile-app/app/screens/AppNavigator/screens/Settings/screens/AboutScreen.tsx

```diff
@@ -245,11 +245,11 @@ function LinkItemRow ({
   }
 
   return (
-    <ThemedTouchableOpacity
+    <ThemedTouchableOpacityV2
       dark={tailwind('bg-mono-dark-v2-00')}
       light={tailwind('bg-mono-light-v2-00')}
       onPress={handlePress}
-      style={tailwind('flex-row px-5 py-4 items-center rounded-lg-v2 mb-2 mx-5')}
+      style={tailwind('flex-row px-5 py-4 items-center rounded-lg-v2 mb-2 mx-5 border-0')}
```

Comment:
This doesn't work on mobile right? `border-0`?

Reply:
it works, `border-none` is the one that would crash on android